### DISCUSSION TITLE
Internet Ports ?

### DISCUSSION THREAD

**Discussion by DanMarquesPT** [Romanitho/Winget-AutoUpdate#303] [General]
Anyone knows which Website need to open on my Firewall to get Intune Auto Update. My client keeps saying "Checking internet connection".

If I use the command line I am able to update , that why I am asking.

**Comment by Romanitho** [comment]
Hi. Nothing special. It tries to reach Github :)

**Reply by DanMarquesPT** [reply]
You 100% correct as after a little started to "Update"

**Reply by Romanitho** [reply]
nice
